tests/general/template_config_test.go

```diff
@@ -0,0 +1,114 @@
+package tests
```

Comment:
mind renaming the file to something like `include_config_source_test.go`?

Reply:
done